src/main/java/org/zalando/nakadi/service/converter/CursorConverterImpl.java

```diff
@@ -43,6 +48,31 @@ public NakadiCursor convert(final SubscriptionCursorWithoutToken cursor)
     }
 
     @Override
+    public List<NakadiCursor> convert(final List<SubscriptionCursorWithoutToken> cursors)
+            throws InternalNakadiException, NoSuchEventTypeException, ServiceUnavailableException,
+            InvalidCursorException {
+        for (final SubscriptionCursorWithoutToken cursor: cursors) {
+            if (null == cursor.getPartition()) {
+                throw new InvalidCursorException(CursorError.NULL_PARTITION, cursor);
+            } else if (null == cursor.getOffset()) {
```

Comment:
both `cursor.getOffset()` and `cursor.getPartition()` are not marked as `@Nullable`. Why have you decided to check it for null here?